app/src/main/java/com/kks/nimblesurveyjetpackcompose/util/extensions/UiExtensioin.kt

```diff
@@ -0,0 +1,12 @@
+package com.kks.nimblesurveyjetpackcompose.util.extensions
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+fun Modifier.loginTextFieldModifier() = this
```

Comment:
Shall we move this into `SplashScreen.kt` since that is the only file that will use this `Modifier`? 😄